lib/elixir/lib/process.ex

```diff
@@ -548,4 +552,16 @@ defmodule Process do
   @compile {:inline, nillify: 1}
   defp nillify(:undefined), do: nil
   defp nillify(other),      do: other
+
+  defp process_register_error(pid, name) do
+    case {alive?(pid), whereis(name)} do
+      {false, _} ->
+        "Process for #{inspect(pid)} is no longer alive"
```

Comment:
How about rewording this to be `"process #{inspect(pid)} is no longer alive"`, would that then be more consistent with the Logger formatting?